app/tests/cases_tests/test_models.py

```diff
@@ -7,14 +7,14 @@
 from django.core.exceptions import MultipleObjectsReturned
 from django.core.files import File
 
+from grandchallenge.cases.utils import get_sitk_image
```

Comment:
This method shouldn't be used in production, could you add a comment in there to say that this is temporary to evaluate this functionality?